src/base/server_type_enum.hpp

```diff
@@ -33,4 +33,34 @@ enum class server_type_config
   PAGE,
   SINGLE_NODE,
 };
+
+inline const char *
+server_type_to_string (SERVER_TYPE type)
+{
+  switch (type)
+    {
+    case SERVER_TYPE_UNKNOWN:
+      return  "unknown";
+    case SERVER_TYPE_TRANSACTION:
+      return "transaction";
+    case SERVER_TYPE_PAGE:
+      return "page";
+    case SERVER_TYPE_ANY:
+      return "any";
+    }
+}
+
+inline const char *
```

Comment:
Not sure, but I think both functions can be made `constexpr` which would help the compiler directly replace the call to function with the result at compile time. There are a few usages where the parameter is a `constexpr`-like expression already. `constexpr` implies `inline const`.